lib/zomato/syncer.rb

```diff
@@ -0,0 +1,50 @@
+module Zomato
+  class Syncer
+    def initialize(restaurants)
+      @restaurants = restaurants
```

Comment:
Same thing, why do we need to hold the restaurants in a class? wouldn't it be better to just create a module?